changelog/16.0/16.0.0/release_notes.md

```diff
@@ -101,11 +102,21 @@ or
 > ALTER TABLE your_table ENGINE=InnoDB;
 ```
 
-#### <a id="vttablet-restore-metrics">VTTablet Restore Metrics
+### <a id="vttablet-restore-metrics">VTTablet Restore Metrics
 
 As part of the VTTablet Sidecar Schema Maintenance Refactor in v16.0.0, we dropped the `local_metadata` table from the sidecar database schema. This table was storing a couple of metrics related to restores from backup, which have now been lost.
 They have been re-introduced in v17.0.0 as metrics that can be accessed from `/debug/vars`.
 
+### <a id="schema-init-upgrade"/>Schema-initialization stuck on semi-sync ACKs while upgrading to `v16.0.0`
+
+During upgrades from `v15.x.x` to `v16.0.0`, as part of `PromoteReplica` call, the schema-init realizes that there are schema diffs to apply and ends up writing to the database.
```

Comment:
This is true when upgrading from v14.x.x and to v16.0.{0,1,2} right?

Reply:
The recommended upgrade path in Vitess is from `n` to `n+1` which is why I only documented `15` to `16`. I will change the phrasing to be more inclusive.